src/index.js

```diff
@@ -268,19 +276,41 @@ function sortClues() {
   return sortedClues[0].pointValue;
 }
 
-function displayDailyDouble(clue, wagerAmount) {
+function displayDailyDouble(clue, wager) {
   let selectedCategory = clueCategories.find(category => category.id === clue.categoryId)
   $('.daily-double-wager').css("display", "flex");
   $('.daily-double-category').text(`${selectedCategory.category.split(/(?=[A-Z])/).join(" ").toUpperCase()}`);
   $('.daily-double-question').text(`${clue.question}`);
+  $('.daily-double-wager-amount').text(`Set your wager between 5 and ${wager} points.`)
+  $('.clue-cards').css("display", "none");
+  $('.selected-clue-info').css("display", "none");
+  $('.game-categories').css("display", "none");
 }
 
-function createDailyDouble() {
-  $('.daily-double-wager-amount').text(`Set your wager between 5 and ${wagerAmount} points.`)
+function evaluateDailyDoubleGuess() {
+  if ($('.daily-double-input').val() === '') {
+    return $('.guess-error').text("Please enter your guess below")
+  } if ($('.daily-double-input').val().toUpperCase() === clue.answer.toUpperCase()) {
```

Comment:
unless this is meant to be an `else if`, I'd put the `if` statement on a new line. It just feels more readable

Reply:
Yes, it should be a new line. Thanks for pointing that out.